test/e2e/utils/runner.go

```diff
@@ -0,0 +1,126 @@
+//go:build e2e
+// +build e2e
+
+package utils
+
+import (
+	"bufio"
+	"errors"
+	"os/exec"
+	"strings"
+
+	gk "github.com/onsi/ginkgo"
+	gm "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/weaveworks/flintlock/api/services/microvm/v1alpha1"
+	"google.golang.org/grpc"
+)
+
+const (
+	containerdBin    = "containerd"
+	flintlockCmdDir  = "github.com/weaveworks/flintlock/cmd/flintlockd"
+	containerdSocket = "/run/containerd-dev/containerd.sock"
+	containerdCfg    = "/etc/containerd/config-dev.toml"
+	grpcDialTarget   = "127.0.0.1:9090"
+)
+
+// Runner is a very poorly named thing and honestly idk what to call it.
+// What it does is compile flintlockd and start containerd and flintlockd.
+// So 'TestSetterUpper' did not sound as slick, but that is what it is.
+// I am happy for literally any suggestions.
+type Runner struct {
```

Comment:
Bootstrapper?

Also not coming up with any decent ideas.

Reply:
there is more coming to it soon, so maybe inspiration will strike